app/src/main/res/values/non_I10n.xml

```diff
@@ -0,0 +1,10 @@
+<resources>
+    <string name="bug_report_url">mzl.la/fxr</string>
+    <string name="settings_key_crash">settings_crash</string>
+    <string name="settings_key_locale">settings_locale</string>
+    <string name="settings_key_telemetry">settings_telemetry</string>
+    <string name="settings_key_session_id">session_id</string>
+    <string name="settings_key_previous_session_id">previous_session_id</string>
+    <string name="private_policy_url">https://www.mozilla.org/en-US/privacy/firefox/</string>
```

Comment:
Don't hard-code en-US in the URL? language negotiation on the server should get you the right localized content.